### ISSUE TITLE
ApexChart Time Series bad rendering in the browser

### ISSUE THREAD

**Issue by Beetix** [Kyvis-Labs/ignition-apexcharts-module#48]
Hi,

I tried the default ApexChart Time Series in a view.

The rendering is fine in the Designer but once in the browser (Firefox and Chrome) the graph is upside down, the axes are inverted:

- Designer

![image](https://github.com/Kyvis-Labs/ignition-apexcharts-module/assets/10374047/898c1569-554c-4862-9b2a-a7be47b953f7)

- Browser
![image](https://github.com/Kyvis-Labs/ignition-apexcharts-module/assets/10374047/0ae687ba-7f9d-4a32-a1f2-bbaf33fdebac)

This was tested on Ignition 8.1.33 with version 1.0.16 of the module.

**Comment by Beetix** [comment]
Closing and reopening the view the Designer actually gives the same rendering issue:
![image](https://github.com/Kyvis-Labs/ignition-apexcharts-module/assets/10374047/9834532b-d823-483c-813c-21ea09935b8f)

**Comment by Beetix** [comment]
It turns out that the axes are inverted due to alphabetical order. The value field name "realistic0" of the data is alphabetically before the timestamp field name "t_stamp". Changing the value field name to for example "u" makes the rendering work.

**Comment by traviscox** [comment]
Yeah I found this out today as well. ApexCharts is looking for the x and y to be in the correct order. We can fix this by either changing the keys to be alphabetical or use arrays:

`[
  {
    "data": [
      [
        1698098270040,
        92.3197320886076
      ],
      [
        1698098285040,
        98.8357416786906
      ],
      [
        1698098300040,
        99.4074424536611
      ],
      [
        1698098315040,
        99.6698195001376
      ],
      [
        1698098330040,
        100.098920176185
      ],
      [
        1698098345040,
        120.147736801713
      ],
      [
        1698098360040,
        93.349219953394
      ],
      [
        1698098375040,
        92.7936729937542
      ],
      [
        1698098390040,
        96.6358322466636
      ],
      [
        1698098405040,
        98.1983313815347
      ],
      [
        1698098420040,
        99.3663180104998
      ],
      [
        1698098435040,
        99.582413164017
      ],
      [
        1698098450040,
        99.9615378800733
      ],
      [
        1698098465040,
        99.8290075001451
      ],
      [
        1698098480040,
        99.8311252090699
      ],
      [
        1698098495040,
        99.858724430026
      ],
      [
        1698098510040,
        100.150820534537
      ],
      [
        1698098525040,
        99.966327143606
      ],
      [
        1698098540040,
        100.126344138819
      ],
      [
        1698098555040,
        99.8460475205365
      ]
    ]
  }
]`

I updated the default example accordingly.
